client/blocks/app-promo/index.jsx

```diff
@@ -138,8 +146,9 @@ export class AppPromo extends Component {
 						height="32"
 						alt="WordPress Desktop Icon"
```

Comment:
I wonder if this `alt` is valuable. IMO it is mostly noise when navigating this button group with a screen reader. WDYT about using an empty `alt` attribute?

Reply:
I appreciate you keeping accessibility in mind, thank you! I've gone ahead to replace this with an empty alt tag in https://github.com/Automattic/wp-calypso/pull/68293/commits/4a9fa624285f54c193fcd488dd859fda5e026efa.